content/logs/index.md

```diff
@@ -4,6 +4,7 @@ kind: Documentation
 autotocdepth: 2
 hideguides: true
 customnav: lognav
+description: "Configure your Datadog agent to gather logs from your host"
```

Comment:
"Configure your Datadog agent to gather logs from your host, containers & services."